spec/features/collection_drafts/forms/invalid_picklists_spec.rb

```diff
@@ -377,15 +377,17 @@
     it 'displays a summary error' do
       within '.summary-errors' do
         expect(page).to have_content('URL Content Type value [badcontenttype] does not match a valid selection option')
-        expect(page).to have_content('Format value [badformat] does not match a valid selection option')
+        # MMT-2425: using kms Format values, means there is no validation
+        # expect(page).to have_content('Format value [badformat] does not match a valid selection option')
```

Comment:
I'm not sure if these lines should be altogether removed or commented out and explained (like so)

Reply:
You can remove the lines. And also, look at the factory it is using, if this value is being populated there you should just remove it.